### ISSUE TITLE
rarbg

### ISSUE THREAD

**Issue by elig0n** [Drugoy/UserScripts#1]
the script isn't working (GreaseMonkey on Firefox), anyone still care to fix it ?

**Comment by Drugoy** [comment]
Yeah, unfortunately they've stripped the catalog/search pages of direct download links + added dynamic (randomly generated) part to them, so this script will never work again.
I've even discussed this with the site owner and he basically said 'screw you', so I'm just going to remove this repo.
:(
